tar/tarsnap.1-man.in

```diff
@@ -883,7 +883,7 @@ With this option, existing hardlinks will be broken, as will any
 symlink that would affect the location of an extracted file.
 .TP
 \fB\-v\fP
-(c, t, x, and list-archives modes only)
+(c, d, t, x, and list-archives modes only)
```

Comment:
Thanks!  `-d -v` only prints the name of each archive that's deleted, while in the other modes it prints each filename (amongst other data).

Could you please add a sentence to this paragraph to that effect?  `In delete mode, tarsnap will...`.